src/app/containers/CpsMetadata/utils.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+// eslint-disable-next-line import/prefer-default-export
+export const getBrandedImage = (image, service) => {
+  // const imageUrl = image
```

Comment:
Can we add an explainer for all the commented out code, or delete it :)

Reply:
It will be deleted shortly